resources/main.mjs

```diff
@@ -260,8 +268,10 @@ class MainBenchmarkClient {
         this._showSection("#details");
     }
 
-    _getFormattedJSONResult() {
+    _getFormattedJSONResult(modern = false) {
```

Comment:
Can we use a dictionary instead so that the call site would be `_formattedJSONResult({modern: true});` instead of `_getFormattedJSONResult(true);`